src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerPlaceholders.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.VisualStudio.Debugger.Metadata
+{
+    // Starting in Visual Studio 2015, Update 1, a new api exists to detect if a type is a function pointer.
```

Comment:
@ericfe-ms Yes for internal dogfooding this is fine. I thought there was a customer scenario that needed to be addressed. We do need to remove this before update1. 
@KevinH-MS I have assigned 5428 to you.